package.json

```diff
@@ -17,9 +18,11 @@
     "node": "^12.16.0"
   },
   "dependencies": {
+    "body-parser": "^1.19.0",
+    "ejs": "^2.5.6",
```

Comment:
I don't think you need this dependency as it is currently not being used

Reply:
ah yes ejs

Reply:
But  body-parser is used here
![image](https://user-images.githubusercontent.com/53966860/117446821-9fafce00-af77-11eb-83a3-b9d8cf56fe2e.png)